Manifests/ManifestsApple/com.apple.system-extension-policy.plist

```diff
@@ -0,0 +1,266 @@
+					<key>pfm_title</key>
+					<string>Team Identifier</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_value_placeholder</key>
+					<string>A1B2C3D4E5</string>
+				</dict>
+			</array>
+			<key>pfm_title</key>
+			<string>Allowed Team Identifiers</string>
+			<key>pfm_type</key>
+			<string>array</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Dictionary of system extensions that will be allowed. Extensions are represented in the dictionary using a team identifier string as key and an array of allowed extension bundle identifier strings as the associated value.</string>
+			<key>pfm_name</key>
+			<string>AllowedSystemExtensions</string>
+			<key>pfm_note</key>
+			<string>Team identifiers cannot be used as keys in this dictionary and at the same time be defined in the "Allowed Team Identifiers" property.</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_name</key>
+					<string>{{key}}</string>
+					<key>pfm_title</key>
+					<string>Team Identifier</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_value_placeholder</key>
+					<string>A1B2C3D4E5</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>{{value}}</string>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_name</key>
+							<string>BundleIdentifier</string>
+							<key>pfm_title</key>
+							<string>Bundle Identifier</string>
+							<key>pfm_type</key>
+							<string>string</string>
+							<key>pfm_value_placeholder</key>
+							<string>com.domainname.applicationname</string>
+						</dict>
+					</array>
+					<key>pfm_title</key>
+					<string>Bundle Identifier</string>
+					<key>pfm_type</key>
+					<string>array</string>
+				</dict>
+			</array>
+			<key>pfm_title</key>
+			<string>Allowed System Extensions</string>
+			<key>pfm_type</key>
+			<string>dictionary</string>
+		</dict>
+		<dict>
+			<key>pfm_description</key>
+			<string>Dictionary of developer teams and their allowed system extension types. Entries use the team identifier as the key and an array of the allowed extension type strings as the associated value.</string>
+			<key>pfm_name</key>
+			<string>AllowedSystemExtensionTypes</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_name</key>
+					<string>{{key}}</string>
+					<key>pfm_title</key>
+					<string>Team Identifier</string>
+					<key>pfm_type</key>
+					<string>string</string>
+					<key>pfm_value_placeholder</key>
+					<string>A1B2C3D4E5</string>
+				</dict>
+				<dict>
+					<key>pfm_name</key>
+					<string>{{value}}</string>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_name</key>
+							<string>SystemExtensionType</string>
+							<key>pfm_range_list</key>
+							<array>
+								<string>DriverExtension</string>
+								<string>NetworkExtension</string>
+								<string>EndpointSecurityExtension</string>
+							</array>
```

Comment:
Please add `pfm_range_list_titles` so we have some more friendly, forward-facing names for the extension types.
```
<key>pfm_range_list_titles</key>
<array>
	<string>Driver</string>
	<string>Network</string>
	<string>Endpoint Security</string>
</array>
```